owner/manager.go

```diff
@@ -171,6 +176,10 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, etcdSession *concurrency.
 	logPrefix := fmt.Sprintf("[%s] %s ownerManager %s", m.prompt, m.key, m.id)
 	var err error
 	for {
+		if err != nil {
+			metrics.CampaignOwnerCounter.WithLabelValues(logPrefix, err.Error()).Inc()
```

Comment:
It seems both `logPrefix` and `err.Error()` are not good label?
Maybe we'll get too much statistic kinds but each with very small count.